addons/mail/static/tests/discuss_app/discuss_tests.js

```diff
@@ -396,8 +404,12 @@ QUnit.test("receive new needaction messages", async () => {
         model: "res.partner",
         res_id: 20,
     });
-    await contains("button:contains(Inbox) .badge");
-    await contains("button:contains(Inbox) .badge", { text: "1" });
+    await contains("button", {
+        containsMulti: [
+            ["div", { text: "Inbox" }],
+            [".badge", { text: "1" }],
+        ],
+    });
```

Comment:
We could (re)-introduce a "text contains" rather than exact match if we want to get rid of them again, but having exact match allowed me to spot few issues in tests across the PRs.

Ideally I'd like to keep exact text, but found only an arbitrary children text node, but I didn't try that yet.